analyses/bgd/README.md

```diff
@@ -40,33 +88,65 @@ The content within this repository is structured as follows:
 │   └── config.yml             <- Parameters to configure the analysis
 │
 ├── trigger_development       <- Scripts related to the trigger analysis
-│
+│   ├── utils             
+│   │   └── utils.py          <- Utiltiy methods for the trigger development scripts and notebooks 
+│   ├── 01_get_glofas_data.py  <- [defunct] Download GloFAS data
```

Comment:
I'd argue that this level of detail here probably isn't necessary and might be tough to maintain. Could probably be fine letting each of the notebook titles speak for themselves?

Reply:
Yeah good point, I'll take these out!